EventListener/Login.php

```diff
@@ -63,6 +67,21 @@ public function setCustomerId($value)
         $this->options['customerId'] = $value;
     }
 
+    protected function getUserIdentifier(TokenInterface $token)
+    {
+        $tokenUser = $token->getUser();
+
+        if ($tokenUser instanceof eZUser || !$tokenUser instanceof UserInterface) {
```

Comment:
I was wondering too, in fact I use the same test as

https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/MVC/Symfony/Security/EventListener/SecurityListener.php#L117

I wanted to be consistent.